lib/zz_main.sh

```diff
@@ -74,14 +80,14 @@ while :; do
       ;;
 
     # FIXME: Please check pacman(8) to see if they are really 2nd operation
-    s|l|i|p|o|m|n)
+    s|l|i|p|o|m|n|g)
       if [[ "$_SOPT" == '' ]]; then
         _SOPT="$_opt"
       else
         if [[ "${_SOPT:0:1}" == "s" ]]; then
           _SOPT="ns"
         else
-          _SOPT="n"
+          _SOPT="$_SOPT$_opt"
```

Comment:
The correct fix for this is https://github.com/icy/pacapt/commit/99ba2c14906bf8335435834d3d032acfbc0b7d94